### DISCUSSION TITLE
Anyway to persist current session as string ?

### DISCUSSION THREAD

**Discussion by Shazoo** [egramtel/tdsharp#134] [Q&A] [answered]
Hi , I used the pyrogram and telethon before .  both of them could persist the login-session to string . 

pyrogram example: 
![image](https://user-images.githubusercontent.com/3198471/178096983-f8306ba0-aa60-470b-9574-3220dd3183cf.png)

Could you please tell me which APIs could help me to get these elements ?
√ means I could get it . 
?  means I need help ..... 

1. dc_id             ？
2. api_id            √
3. test_mode     √
4. auth_key        ？
5. user_id           √
6. is_bot             √

**Comment by ForNeVeR** [comment, marked as answer]
Sorry, but I guess that's a question for the upstream developers. Try asking there: https://github.com/tdlib/td/issues
